uli-website/src/pages/blog/chisel-online-abuse.mdx

```diff
@@ -0,0 +1,33 @@
+date: 02-05-2024
+tags: 
+---
+
+import ContentPageShell from "../../components/molecules/ContentPageShell.jsx"
+
+<ContentPageShell>
+
+![](/HeroIllustration.gif)
+
+Tackling online gender-based violence is a large issue – with policy and legal, behavioural change, tackling patriarchal societal norms, and political will – as just some of the components that need to be addressed.  
+Our question was how could we help address the problem from a machine learning lens; and what did it mean to do that in a participatory manner? 
+Uli, or chisel in Tamil, became our collective effort to help chisel away at this problem. Early input into possible solutions emphasised the importance of a simple and lightweight tool, easy to use and adaptable. 
+On the front end, the result was the Uli plug-in which allows people to track and archive problematic content (useful also for analysing patterns and reporting to law enforcement), as well as redact slurs. It works primarily on X (formerly Twitter) and on several pages of the Chrome and Brave browsers. 
+At the same time, the backend of the process and the approach to its design was as important as the tool that emerged. There was a concerted effort to build in feminist principles and to avoid replicating neo-colonial mindsets. 
+
+In fact, the design to redact the slurs was inspired by the design of a quilt. The quilt-making process has always reflected intimate emotional labour, gendered histories and storytelling of the world around us. Like quilts, Uli was made collaboratively, and are both personal and communal objects that empower and celebrate.
+In taking this approach, there were five main principles that we tried to integrate into the design process. 
+The first and second were intersectionality and a community-based approach. Uli was born from the collective labour of journalists, activists, community influencers, and writers who are engaged in the struggle against the interwoven caste, religion, gender and sexuality-based violence faced both online and offline. Not only did this reflect a diversity of backgrounds beyond gender, but what constitutes abuse was also determined by those targeted online. (Watch for more in this in Blog 4 of this series.)
+The third was the visibility of labour. We made sure that contributing partners and annotators were mentioned on the website and included on the technical papers. 
+The fourth was a localised approach – a tool developed by a team in India for Indians. 
+Finally, the fifth was pluralism. We created some features for customization so people can alter it around their experiences. This includes their privacy requirements. All features, whenever possible, have a version that can be used locally. In other words, it happens on the users’ machine and no data is sent out. Where data has to be shared, all personal information is stored in an encrypted format. Plus, since the code is open source, anyone can tweak the model and adapt it for their own context. 
```

Comment:
"tweak the model"  should be changed to "tweak the source code"

Reply:
changing it!